src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas2/ApplicationServiceTest.kt

```diff
@@ -125,22 +127,23 @@ class ApplicationServiceTest {
 
   @Nested
   inner class GetApplicationsWithPrisonCode {
```

Comment:
are we missing some unit test coverage of `getApplications` here?
It looks like this block tests with prison code, but is there a test for when prison code is null?
I wonder if we tightened up the unit tests around this method if we could look at paring back the integration tests in the future...

Reply:
Added a more recent commit 68396fd0028cd9bf36471e836afbcd3ad1b612c2 with these tests in.